### DISCUSSION TITLE
[Feature Idea] Setting to limit the number of database backups kept

### DISCUSSION THREAD

**Discussion by jon-f-novabackup** [usememos/memos#2173] [Ideas]
Hi is there a setting, like to place in the .env (docker) file that can controls how many database backups to keep in Memos v0.14.4? I can see that it is in fact backing up the DB to my ~./memos/ folder, and it has created 1 backup per day ever since I enabled that option 16 days ago. Is there a limiter for how many backups to keep or does it have a built in limit (that I haven't hit yet after 16 days) and if so what is that default number of backups to keep? My backup setting in the UI is 86400 seconds (so one backup every 24 hours) and I have about 16 of those backed up DB files in ~./memos/ one per each day (the date shows in the filename in fact) for reference; and I think that corresponds to the day that I enabled that backup setting as to how many days of backups it has kept so far. If we use that option (and we care about disk space usage) are we supposed to do manual cleanup of the existing DB backup files in that case, via the shell or a script, or is there a feature request for the UI to be able to set the limit of number of backups to keep, that is planned?

This was discussed earlier in the Discord #question group on 08/21/2023:
![image](https://github.com/usememos/memos/assets/50121594/996335a2-0daf-48f1-bdf0-f349e0eb7af3)

**Comment by boojack** [comment]
Maybe we can refer to the UI design from @pocketbase.

<img width="806" alt="image" src="https://github.com/usememos/memos/assets/24653555/551bb2e6-2184-4694-8f9f-7a3a7b130331">
